### DISCUSSION TITLE
Is Starship Code signed and Notarized?

### DISCUSSION THREAD

**Discussion by esarella** [starship/starship#1689] [Q&A] [answered]
Hello starship Devs, 

I'm trying to get starship to be a part of our enterprise toolset. For that to happen I have to get approval from our security team. For MacOS apps they want the app to be Code signed and Notarized as described here https://developer.apple.com/documentation/xcode/notarizing_macos_software_before_distribution wondering if and when this could happen?

**Comment by matchai** [comment, marked as answer]
We haven't yet notarized Starship, but it should certainly be doable!
If you'd like to see it done, please create a feature request issue so that we can track the task. 😄

**Reply by esarella** [reply]
Thank you. I've submitted a new feature request issue to notarize the app
